src/main/java/org/sagebionetworks/web/client/widget/modal/Dialog.java

```diff
@@ -95,6 +101,28 @@ public void configure(String title, Widget body, String primaryButtonText, Strin
 	}
 	
 	/**
+	 * @param title The text shown in the title bar.
+	 * @param primaryButtonText The text for the primary button (i.e "Save").  The primary button is highlighted.
+	 * @param defaultButtonText The text for the default button (i.e "Cancel").  The default button will not be highlighted.  If null, will hide default button.
+	 * @param callback
+	 * @param autoHide if true, will hide the dialog on primary or default button click.
+	 */
+	public void configure(String title, String primaryButtonText, String defaultButtonText, Callback callback, boolean autoHide) {
```

Comment:
This is the exact same configure as above without clearing out the body of the container.